security/roles/secure_login_node/tasks/configure_alerting.yml

```diff
@@ -16,16 +16,43 @@
 - name: Include local_repo_access.yml file
   ansible.builtin.include_vars: "{{ local_repo_access_path }}"
 
-- name: Install mailx and postfix
+- name: Install mailx and postfix for Rhel/Rocky
```

Comment:
RHEL

Reply:
Addressed changes in internal PR. It will be available in next set of PRs